wails.json

```diff
@@ -11,7 +11,7 @@
   "info": {
     "companyName": "Gobalsky Labs Ltd.",
     "productName": "Vega Wallet Dev",
-    "productVersion": "0.11.4",
+    "productVersion": "0.11.0",
```

Comment:
Should be 0.12.0

And there are other things to change. see the RELEASE documentation in the repo for the version update process.

Reply:
Wasn't meant to be in here, will revert. I'll make branch for releasing v0.12 tomorrow